docs/DeveloperGuide.adoc

```diff
@@ -780,19 +780,27 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 [appendix]
 == Product Scope
 
-*Target user profile*: Couples in NUS who:
-
-* want to plan dates, in and out of school
-* want to store their memories via private journal entries
+*Target user profile:*
+
+* is currently a student of NUS
+* is in a relationship with a student of NUS
+* may have to travel to unfamiliar locations in NUS
+* want to remember special moments
+* want to remember special dates
+* want to schedule meetings
 * prefer desktop apps over other types
 * can type fast
 * prefer typing over mouse input
 * are reasonably comfortable using CLI apps
 
-*Value proposition*: Help couples in NUS to create new memories and remember them through:
+*Value proposition:* all-in-one desktop app to help NUS couples make and remember memories with each other
+
+*Feature contribution:*
 
-* Providing features such as NUS timetable comparator to allow couples in NUS to plan dates more easily
-* Jot down journal entries to remember their dates
+* Timetable viewer: To help couples identify common breaks so they can plan meetings
```

Comment:
Maybe can emphasize that the timetable helps to identify common breaks during school term? because if not it's similar to scheduler

Reply:
I meant it as purpose + link to our project scope. I think it shows how our features can work together. One identifies common breaks, the other allows you to schedule a meeting during that common break? What do the others think?

Reply:
I think for the Timetable view simply just put help to identify common breaks and remove "so they can plan meetings", then insert that mean "so they can plan meetings" into schedular. So to be distinct.